shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapterTest.java

```diff
@@ -246,11 +258,13 @@ public void alterPartitionEvent() {
     MetaStoreEvent actual = messageReaderAdapter.read().get();
 
     assertMetaStoreEvent(expected, actual);
+    verify(messageReader, times(1)).delete(RECEIPT_HANDLE);
```

Comment:
times(1) is the default ok to do: `verify(messageReader)`

Reply:
Thanks, also much cleaner